packages/manager/apps/key-management-service/src/pages/listing/terminate/TerminateKms.tsx

```diff
@@ -33,12 +35,21 @@ export default function TerminateKms() {
     });
     closeModal();
     clearNotifications();
-    addSuccess(
-      t('key_management_service_terminate_success_banner', {
-        ServiceName: okmsId,
-      }),
-      true,
-    );
+    if (ovhSubsidiary === 'US') {
+      addSuccess(
+        t('key_management_service_terminate_success_banner_us', {
+          ServiceName: okmsId,
+        }),
+        true,
+      );
+    } else {
+      addSuccess(
+        t('key_management_service_terminate_success_banner', {
+          ServiceName: okmsId,
+        }),
+        true,
+      );
+    }
```

Comment:
```suggestion
      addSuccess(
        t((ovhSubsidiary === 'US' ? 'key_management_service_terminate_success_banner_us' : 'key_management_service_terminate_success_banner'), {
          ServiceName: okmsId,
        }),
        true,
      );
```

Juste as suggestion for next time